tools/automator/utils.sh

```diff
@@ -42,3 +42,8 @@ split_on_commas() {
 evaluate_tmpl() {
   bash -c "eval echo \"$1\""
 }
+
+hash() {
+  local val="$1" chars="${2:-8}"
+  echo -n "$val" | md5sum | cut -c1-"$chars"
```

Comment:
no need for echo

```bash
md5sum <<< "$1"
```

Reply:
`<<<` is not POSIX-compliant so I prefer to avoid it where possible.

Reply:
Check the `#!` this is a `bash` program, not `sh`

Reply:
Thanks. I know the shebang is `bash` in this case. It is just a scripting habit for maximal portability. As `<<<` is only available in newer bash version (3+ I believe).